basc_archiver/sites/fourchan.py

```diff
@@ -34,14 +34,14 @@
 
 # new urls
 #FOURCHAN_API = 'api.' + FOURCHAN # api.4chan.org also works, but 4cdn still on
-FOURCHAN_IMAGES = 'is.' + FOURCHAN
-FOURCHAN_THUMBS = 'is.' + FOURCHAN
+#FOURCHAN_IMAGES = 'is.' + FOURCHAN
+#FOURCHAN_THUMBS = 'is.' + FOURCHAN
 #FOURCHAN_STATIC = 's.' + FOURCHAN_CDN # static.4chan.org also works, but not yet
 
 # cdn domains (no longer in use for images)
```

Comment:
Nice. If this PR's correct then we can also remove the `(no longer in use for images)` from this comment :)